server/api/resolvers/donationFormResolvers.ts

```diff
@@ -18,7 +18,30 @@ const donationFormQueryResolvers = {
     },
     donationForms: async (_, __, ___): Promise<Array<DonationFormInterface>> => {
         return DonationForm.find().exec();
-    }
+    },
+    donationFormsPage: async (_, { skip, limit, filterOptions }, __): Promise<Array<DonationFormInterface>> => {
+        const { name, formType, status } = filterOptions;
+        let filter: any = {}
+        if (name) {
+            filter.name = { $regex: "^" + name + ".*", $options: "i" };
+        } 
+        if (formType) {
+            if (formType == "GENERAL"){
+                filter.requestGroup = { $eq: null };
+            }
+            if (formType == "SPECIFIC"){
+                filter.requestGroup = { $ne: null };
+            }
+        }
+        if (status){
+            filter.status = status;
+        }
+        return DonationForm.find(filter)
+            .sort({ name: "ascending", _id: "ascending" })
```

Comment:
Is it possible to just sort ascending by name?

Reply:
Done